modules/microshift-manifests-upgrade-scenario-rpm.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-upgrade-scenario-rpm_{context}"]
+== Manifest upgrade scenario for RPM systems
+Use the following procedure in the upgrade scenario for RPM systems where some resources need to be removed while retaining others to preserve data.
+
+.Procedure
+
+. Identify the manifest that requires updating.
+. Create new manifests that needs to be applied in the manifest path.
+. Create new manifests for resource deletion. It is not necessary to include the `spec` in these manifests. See link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{ocp-version}/html/running_applications/applications-with-microshift#microshift-applying-manifests-example_applications-microshift[Using manifests example] to create new manifests using the example.
```

Comment:
see previous comment about links to internal content.

Reply:
done as per comment https://github.com/openshift/openshift-docs/pull/83246#discussion_r1817012102.